imagersite/imager_images/tests.py

```diff
@@ -314,3 +311,44 @@ def test_edit_album_view(self):
         response = view(req)
         print(response.status_code)
         self.assertTrue(response.status_code == 200)
+
+
+class PaginationTests(TestCase):
+    """Test pagination."""
+
+    def setUp(self):
+        """User setup for tests."""
+        self.client = Client()
+        self.request = RequestFactory()
+        self.users = UserFactory.create()
+        self.images = [ImageFactory.create() for i in range(10)]
+        self.albums = [AlbumFactory.create() for i in range(10)]
+
+    def test_photo_paginator(self):
+        """Test that 10 photos in groups of 4 create 3 pages."""
+        paginator = Paginator(self.images, 4)
+        self.assertEqual(10, paginator.count)
+        self.assertEqual(3, paginator.num_pages)
+        self.assertEqual([1, 2, 3], list(paginator.page_range))
```

Comment:
You don't have to test that Django's paginator object works. Assume that Django works the way that it's supposed to. Instead, test the code you've written.